apis/coursebatchmanapi/coursebatchmanapi.yaml

```diff
@@ -56,7 +56,843 @@ paths:
+        description: Extra data can be passed in query param with above mention key and comma separated value:Example ,?orgdetails=email,name . Sunbird will take this query param and pass to content service as it is, and content service knows what need to be done.
+        type: string
+        required: false 
+      - name: Body
+        in: body
+        required: true
+        description: >-
+            The body refers to the format of the request.
+              - The body contains metadata about the batches to be listed
+        schema:
+         allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/CourseBatchSearchReq'
+      
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description: >-
+            The Content Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'The time stamp identifying when the **List the existing batches** request was sent. '
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'It is a Unique ID which identifies the request incase the same API is executed multiple times.'
+      - name:  x-authenticated-user-token
+        in: header
+        required: true
+        type: string
+        description: 'The alphanumeric string for accessing the API'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'SUCCESS. The **Lists the existing batches** operation was successful!'
+          schema:
+            allOf:
+              - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+              - $ref: '#/definitions/SearchBatchResponse'
+        '400':
+          description: 'BAD REQUEST,**Lists the existing batches** operation failed !The possible reason for failure is that you may have missed providing input for a mandatory parameter'
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/BatchCreateErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR. We track these errors automatically and try to set it right at the earliest. Try refreshing the page. If the problem persists contact us at info@sunbird.org.'

```

Comment:
INTERNAL SERVER ERROR, we are tracking the error and cater to it on priority. Try to refresh the page and if the problem persists contact us at info@sunbird.org